day_7/10_speckled_frogs3.rb

```diff
@@ -0,0 +1,12 @@
+p "Input number of froggy friends:"
+frog = gets.chomp.to_i
+
+while frog > 1
+  p "#{frog} speckled frogs sat on a log \n eating some most delicious bugs. \n One jumped in the pool where it's nice and cool,\n then there were #{frog - 1} speckled frogs. \n"
+  frog -= 1
+end
+
+while frog == 1 #correct for plurality
```

Comment:
It's a bit of an antipattern to say while at one specific value. One way to avoid this would have been to have a nested if/else statement inside of your while loop to evaluate which wording to use.